packages/account-v2/src/components/Dropzone/Dropzone.tsx

```diff
@@ -103,7 +103,7 @@ const Dropzone: React.FC<TProps> = ({
         <div {...getRootProps()} className={className} ref={rootRef as RefObject<HTMLDivElement>}>
             <input
                 {...(getInputProps() as DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>)}
-                data-testid='dt_dropzone-input'
+                data-testid='dt_dropzone_input'
```

Comment:
@suisin There are more data testids in this file with worng format, can you check them please